package.json

```diff
@@ -18,6 +18,7 @@
     "build:font-awesome": "node scripts/build-fontawesome.js",
     "build:libs": "uglifyjs client/js/libs/*.js client/js/libs/jquery/*.js client/js/libs/handlebars/*.js -o client/js/libs.min.js --source-map client/js/libs.min.js.map --source-map-url libs.min.js.map -p relative",
     "build:handlebars": "handlebars client/views/ -e tpl -f client/js/lounge.templates.js",
+    "build:postcss": "postcss -u postcss-cssnext -m file -o client/css/build.css client/css/style.css",
```

Comment:
Oh, so it is. 

Alright, so are you waiting till webpack is merged before this gets done?